src/stylesheets/components/_alerts.scss

```diff
@@ -1,3 +1,17 @@
+// Alert variables ---------- //
+// scss-lint:disable TrailingWhitespace
```

Comment:
@carlosvalle we are no longer using `scss-lint` so this line should be a-okay to remove.

See https://github.com/18F/web-design-standards/commit/95d626ad346748a71c9260514fae0771d83e22d3 for details on those changes.